woocommerce-services.php

```diff
@@ -379,6 +382,10 @@ public function set_rest_self_help_controller( WC_REST_Connect_Self_Help_Control
 			$this->rest_self_help_controller = $rest_self_help_controller;
 		}
 
+		public function set_rest_service_data_refresh_controller( WC_REST_Connect_Service_Data_Refresh_Controller $rest_service_data_refresh_controller ) {
```

Comment:
Not sure the point of storing a reference to WC_REST_Connect_Service_Data_Refresh_Controller is here. I know we do it with other classes but usually that is done for purposes of adding mocks for unit tests. Since we're not doing that lets just remove the rest_service_data_refresh_controller property and set_rest_service_data_refresh_controller method and stop calling it below.

Reply:
Yeah, all those `set_*` seemed useless to me too - again, I added it because I didn't want to create pushback, but it seemed to have created the opposite effect 😅  removed this instance in d3391a17